src/schema/sale_artwork.js

```diff
@@ -272,6 +261,42 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLFloat,
         deprecationReason: "Favor `minimum_next_bid`",
       },
+      my_increments: {
+        type: new GraphQLList(BidIncrementsFormatted),
+        resolve: async (
+          { _id, minimum_next_bid_cents, sale_id, symbol },
+          options,
+          request,
+          { rootValue: { incrementsLoader, lotStandingLoader, saleLoader } }
+        ) => {
+          const myLotStandings = await lotStandingLoader({
+            sale_artwork_id: _id,
+          })
+
+          const myCurrentMax = get(
+            myLotStandings,
+            "0.max_position.max_bid_amount_cents"
+          )
+
+          const myMinimumNextBid = myCurrentMax
+            ? maxBy([myCurrentMax, minimum_next_bid_cents])
+            : minimum_next_bid_cents
+
+          return bid_increments_calculator({
+            sale_id,
+            saleLoader,
```

Comment:
Also, it won't let me comment so far above but when we fetch information about the sale artwork (specifically, the `minimum_next_bid_cents`), we'll want to grab that _ignoring_ metaphysics' cache. I don't think there's a way to specify this right now and I believe it's going to come up in the FE meeting tomorrow. (so not blocking, just a note)

Reply:
Can you describe more the caching you expect? Maybe this should use an auth'ed loader (or the 'one off' factory type like for system time?).

Alternatively, we could enhance the 'without authentication' factory code to accept `{ cache: false }` and have it skip that.

I am curious about the flow of events however that you want this to be uncached from Metaphysics, but, presumably cached in Gravity? (If it's cached in gravity per user then we can just use an auth'ed loader here. If it's not cached per user I'm confused why you specifically want to skip MP's cache).

Reply:
The flow of events (where it goes wrong) is that during the bidding flow, after you place a bid you may have been immediately outbid and will have to place another one. At that point, you can click back to the "select my max bid" screen which will refresh the increments and _should_ show you the correct next bid.

At that point, the sale artwork's cache will have been invalidated in gravity, but the first request to MP will return a stale value before it's updated. So we'll have the wrong `minimum_next_bid_cents` from which to compare. This behavior will become better when we're adding the call to lot standings, but it's still not ideal.

I think your suggestion to enhance the factory is the way to go (but I won't do that in this PR).